packages/mongodb-chatbot-ui/tsconfig.json

```diff
@@ -18,7 +20,9 @@
     "noFallthroughCasesInSwitch": true,
     "noImplicitAny": true,
     "noImplicitReturns": true,
-    "noImplicitThis": true
+    "noImplicitThis": true,
+
+    "types": ["vitest/globals", "@testing-library/jest-dom"]
   },
-  "include": ["./src/**/*.ts", "./src/**/*.tsx"]
+  "include": ["./src/**/*.ts", "./src/**/*.tsx", "./vitest.setup.ts"],
```

Comment:
is this saying that test set up should be included in the bundle?

Reply:
Yeah it is - not sure why I included it but I don't think it's necessary